composeApp/src/commonMain/composeResources/values/strings.xml

```diff
@@ -0,0 +1,2 @@
+<resources>
```

Comment:
Are we able to have string resources here as well? Or is the file totally overriden by the ooni and dw variants?

Reply:
We can do so right now. I tested locally but it fails on the ci.